cspell.json

```diff
@@ -36,7 +36,26 @@
     "ADMGR",
     "coef",
     "structs",
-    "tlsv"
+    "tlsv",
+    "Slava",
+    "Balasanov",
+    "Farterson",
+    "Bacor",
+    "simondlr",
+    "marketcap",
+    "mstore",
+    "Enjin",
+    "newuri",
+    "funciton",
+    "delegatecall",
+    "diamod",
+    "Ubistick",
```

Comment:
This should be UbiquiStick. Perhaps add "Ubiqui" as a word since Stick already is.